app/src/lib/common/types/ComponentTypes/VisitorCommentType.ts

```diff
@@ -0,0 +1,44 @@
+export interface VisitorCommentPropsType {
+  id?: string;
+  commentList?: VisitorCommentListType[];
+  theme?: VisitorCommentThemeType;
+  backgroundColor?: string;
+  buttonColor?: string;
+  commentInputProps?: CommentInputPropsType;
+  desPlaceholder?: string;
+  nicknamePlaceholder?: string;
+  passwordPlaceholder?: string;
+  inputBackgroundColor?: string;
+  userInputLineColor?: string;
+  progressbarColor?: string;
+  isShowScrollDownIcon?: boolean;
+  scrollDownIconColor?: string;
+}
+
+export interface VisitorCommentListType {
+  des?: string;
```

Comment:
```suggestion
  description?: string;
```

I dont know the name called des.
What is des?
I think ‘desctiprion’ is better than ‘des’.
 ‘des’ is not a common abbreviation. People can be confused about what ‘des’ is.